app/scripts/metamask-controller.js

```diff
@@ -503,7 +507,6 @@ module.exports = class MetamaskController extends EventEmitter {
       // provider approval
       approveProviderRequestByOrigin: providerApprovalController.approveProviderRequestByOrigin.bind(providerApprovalController),
       rejectProviderRequestByOrigin: providerApprovalController.rejectProviderRequestByOrigin.bind(providerApprovalController),
-      forceApproveProviderRequestByOrigin: providerApprovalController.forceApproveProviderRequestByOrigin.bind(providerApprovalController),
```

Comment:
This function is still referenced in `actions.js`, as well as in `ui/app/pages/home/home.component.js` in the default props. It should be removed from those two places - it doesn't appear to be used anymore.

Reply:
Good catch, thanks